### PR TITLE
Scar and Tattoo Fixes

### PR DESCRIPTION
**Pull request by Aikakakah**
## About the PR
Fixed the issue of chest tattoos showing through clothes. Includes a fix for the back of the Nightling chest tattoo showing on the back of the neck on males and all chest tattoos and scars showing through clothing on females.

## Why / Balance
I saw it. It annoyed me. I fixed it.

## Technical details
Added male and female variations of affected scars/tattoos.

## Media
Green Nightling tattoo on female.
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- fix: Fixed tattoos and scars showing through clothing.

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml [hunk 1]
```diff
@@ -41,11 +41,23 @@
     state: scar_top_surgery_long
 
 - type: marking
-  id: ScarChest
+  id: ScarChestMale
```
Comment (ghost) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml]: `#DeltaV: Splitting the scars and tattoos`

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml [hunk 2]
```diff
@@ -41,11 +41,23 @@
...
   bodyPart: Chest
   markingCategory: Chest
   speciesRestriction: [Human, Dwarf, Felinid, Oni] #Einstein Engines - Felinid, Oni
+  sexRestriction: [Male]
```
Comment (ghost) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml]: `#DeltaV: Splitting the scars and tattoos`

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml [hunk 3]
```diff
@@ -41,11 +41,23 @@
...
+  followSkinColor: true
+  sprites:
+  - sprite: Mobs/Customization/scars.rsi
+    state: scar_chest_male
+
+- type: marking
+  id: ScarChestFemale
```
Comment (ghost) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml]: `#DeltaV: Splitting the scars and tattoos` or you could split it into a DeltaV file if it's almost an entirely new addition in an upstream file.

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml [hunk 4]
```diff
@@ -1,30 +1,62 @@
 - type: marking
-  id: TattooHiveChest
+  id: TattooHiveChestMale
```
Comment (ghost) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: `#DeltaV: Splitting the scars and tattoos`

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml [hunk 5]
```diff
@@ -1,30 +1,62 @@
...
         !type:TattooColoring
       fallbackColor: "#666666"
   sprites:
   - sprite: Mobs/Customization/tattoos.rsi
-    state: tattoo_hive_chest
+    state: tattoo_hive_chest_male
 
 - type: marking
-  id: TattooNightlingChest
+  id: TattooHiveChestFemale
```
Comment (ghost) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: `#DeltaV: Splitting the scars and tattoos`

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml [hunk 6]
```diff
@@ -1,30 +1,62 @@
...
+    default:
+      type:
+        !type:TattooColoring
+      fallbackColor: "#666666"
+  sprites:
+  - sprite: Mobs/Customization/tattoos.rsi
+    state: tattoo_nightling_male
+
+- type: marking
+  id: TattooNightlingChestFemale
```
Comment (ghost) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: Could probably comment the original nightling tattoo `#DeltaV: Splitting the scars and tattoos` and then put the new female one in a DeltaV file

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml [hunk 7]
```diff
@@ -1,30 +1,62 @@
...
       type:
         !type:TattooColoring
       fallbackColor: "#666666"
   sprites:
   - sprite: Mobs/Customization/tattoos.rsi
-    state: tattoo_nightling
+    state: tattoo_hive_chest_female
+
+- type: marking
+  id: TattooNightlingChestMale
```
Comment (ghost) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: `#DeltaV: Splitting the scars and tattoos`

**Review by ghost — COMMENTED** [review]
All this looks like is splitting of the markings - most of which touches upstream files. I'd say maybe go fix this in an upstream PR (at space-wizards/space-station-14), - that might take a while though. If you wanna get this accepted here though, you're gonna have to like heavily split it so that things are either DeltaV commented or make a few DeltaV files for tattoos.

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml (hunk 6 above)
Comment (Aikakakah) [reply Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: It's only removing 1-2 pixels, wouldn't say it's different enough to warrant the DeltaV branding on it. Will comment it all accordingly.

Comment (MilonPL) [reply Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: It's not about "DeltaV branding", it's about stating changes compared to the upstream.

Comment (Aikakakah) [reply Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: Ohh that makes more sense, I thought it was for new content (still getting used to this 😅)

**Comment by Aikakakah** [comment]
Edited it further to keep all original files in their respective folder, new changes are in the DeltaV folder.

File: Resources/Locale/en-US/markings/scars.ftl [hunk 8]
```diff
@@ -12,3 +12,6 @@ marking-ScarTopSurgeryLong = Top Surgery Scar (Long)
 
 marking-ScarChest-scar_chest = Chest Scar
 marking-ScarChest = Chest Scar
+
+marking-ScarChestFemale-scar_chest_female = Chest Scar
+marking-ScarChestFemale = Chest Scar
```
Comment (MilonPL) [inline Resources/Locale/en-US/markings/scars.ftl]: Move to deltav namespace

File: Resources/Locale/en-US/markings/tattoos.ftl [hunk 9]
```diff
@@ -1,8 +1,14 @@
 marking-TattooHiveChest-tattoo_hive_chest = Back Tattoo (Hive)
 marking-TattooHiveChest = Back Tattoo (Hive)
 
-marking-TattooNightlingChest-tattoo_nightling = Chest Tattoo (nightling)
-marking-TattooNightlingChest = Chest Tattoo (Nightling)
```
Comment (MilonPL) [inline Resources/Locale/en-US/markings/tattoos.ftl]: Leave this alone, move the female variants to deltav namespace

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml [hunk 10]
```diff
@@ -45,7 +45,19 @@
...
+- type: marking
+  id: ScarChestFemale #DeltaV: Splitting the scars and tattoos
+  bodyPart: Chest
+  markingCategory: Chest
+  speciesRestriction: [Human, Dwarf, Felinid, Oni] #Einstein Engines - Felinid, Oni
+  sexRestriction: [Female] #DeltaV: Splitting the scars and tattoos
+  followSkinColor: true
+  sprites:
+  - sprite: DeltaV/Mobs/Customization/scars.rsi
+    state: scar_chest_female
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml]: At least comment with "#Start DeltaV additions" and "#End DeltaV" or ideally move to deltav namespace

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml [hunk 11]
```diff
@@ -13,18 +14,49 @@
...
   coloring:
     default:
       type:
         !type:TattooColoring
       fallbackColor: "#666666"
   sprites:
-  - sprite: Mobs/Customization/tattoos.rsi
-    state: tattoo_nightling
+  - sprite: DeltaV/Mobs/Customization/tattoos.rsi
+    state: tattoo_hive_chest_female
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: deltav comments

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml [hunk 12]
```diff
@@ -13,18 +14,49 @@
...
+  speciesRestriction: [Human, Dwarf, Felinid, Oni] # Delta V - Felinid, Oni
+  sexRestriction: [Female] #DeltaV: Splitting the scars and tattoos
+  coloring:
+    default:
+      type:
+        !type:TattooColoring
+      fallbackColor: "#666666"
+  sprites:
+  - sprite: DeltaV/Mobs/Customization/tattoos.rsi
+    state: tattoo_nightling_female
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml]: Move or comment changes

**Review by MilonPL — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/markings/tattoos.ftl (hunk 9 above)
Comment (Aikakakah) [reply Resources/Locale/en-US/markings/tattoos.ftl]: There's a modification to the male one, it had a pixel peeking past the body. Should I move both variants?

**Review by MilonPL — APPROVED** [review]
forgor

**Pull request merged**